### DISCUSSION TITLE
Object Array in prisma without relationship

### DISCUSSION THREAD

**Discussion by Xazgl** [prisma/prisma#18223] [Q&A]
Is it possible to save an array of objects in one field without using links ?

![image](https://user-images.githubusercontent.com/86427773/223030526-712bc60a-4bd6-4749-9814-b6c95bbf1661.png)
![image](https://user-images.githubusercontent.com/86427773/223030603-789bcf51-7bc8-4c54-a686-0eb1d3b94780.png)

**Comment by nurul3101** [comment]
Hello @Xazgl 👋 

Can you please elaborate on what you mean by 

> without using links

What is the value of `adminFromReq.drivers`?

**Reply by nurul3101** [reply]
@Xazgl Did you had a chance to look at my response? Do let us know in case you still need any assistance. It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
@Xazgl Do let us know in case you need any further assistance.
